src/core/undo/ReflectUndoAction.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Xournal++
+ *
+ * Undo action for rotate (EditSelection)
```

Comment:
```suggestion
 * Undo action for reflection (EditSelection)
```

Reply:
Forgot to change this. I'll update it in the next round of changes.